components/hab/src/cli.rs

```diff
@@ -367,16 +358,17 @@ impl FromStr for KeyType {
     }
 }
 
-////////////////////////////////////////////////////////////////////////
-
-fn arg_cache_key_path() -> Arg<'static, 'static> {
-    Arg::with_name("CACHE_KEY_PATH").long("cache-key-path")
-                                    .validator(util::non_empty)
-                                    .env(CACHE_KEY_PATH_ENV_VAR)
-                                    .default_value(&*CACHE_KEY_PATH_DEFAULT)
-                                    .help("Cache for creating and searching for encryption keys")
+impl fmt::Display for KeyType {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            KeyType::Public => write!(f, "Public"),
+            KeyType::Secret => write!(f, "Secret"),
```

Comment:
Unless there's a clear reason otherwise, I think it's good practice for `Display` and `FromStr` to be inverses of each other. That is, the following should hold: 

```rust
assert_eq!(original_string, original_string.parse::<MyType>()?.to_string()?);
```

Here, that's just a long way of saying that `"Public"` and `"Secret"` should be lower-case :sweat_smile:

Reply:
😅